src/checkbox/checkbox.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+
+class Checkbox extends Component {
+  render() {
+    const { className = '', style, ...props } = this.props;
+    return (
+      <div className={`mt-checkbox ${className}`} style={style}>
+        <input className="mt-checkbox_input" type="checkbox" {...props} />
```

Comment:
It is not obvious that we would put `style` on the `div` and not the `input`, while everything else goes on the `input`. We may want to document that in the `props` table

Reply:
Documented!